fdbclient/include/fdbclient/FDBTypes.h

```diff
@@ -750,11 +750,38 @@ struct GetRangeLimits {
 
 struct RangeResultRef : VectorRef<KeyValueRef> {
 	constexpr static FileIdentifier file_identifier = 3985192;
-	bool more; // True if (but not necessarily only if) values remain in the *key* range requested (possibly beyond the
-	           // limits requested) False implies that no such values remain
-	Optional<KeyRef> readThrough; // Only present when 'more' is true. When present, this value represent the end (or
-	                              // beginning if reverse) of the range which was read to produce these results. This is
-	                              // guaranteed to be less than the requested range.
+	bool more; // True if values remain in the *key* range requested (possibly beyond the
+	           // limits requested), but not necessarily only if, for example, in getRangeStream(), 'more' is always set
+	           // to true, as each stream fragment doesn't know if it's the last one. Instead, it uses
+	           // `error_code_end_of_stream` for the end.
+	           // False implies that no such values remain
```

Comment:
```suggestion
	// True if the range may have more keys in it (possibly beyond the specified limits).
	// More can be true even if there are no keys left in the range, e.g. if a shard boundary is hit.
	// Additionally, getRangeStream() always sets 'more' to true and uses the 'end_of_stream'
	// error to indicate that a range is exhausted. If 'more' is false, the range is guaranteed to
	// have been exhausted.
	bool more;
```

Reply:
Thanks for adding that. Updated with a little bit more change.